tests/DCPS/LivelinessTest/dualpub.cpp

```diff
@@ -0,0 +1,367 @@
+#include "Writer.h"
```

Comment:
Collapse dualpub, dualsub, pub, and sub into a single executable.
There should be a single run_test.pl.